src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -367,11 +368,37 @@ def load_params(command):
         logger.debug("Unable to load commands for '%s'. No module in command module map found.",
                      command)
         return
-    module_to_load = command_module[:command_module.rfind('.')]
+    last_dot_index = command_module.rfind('.')
+    if last_dot_index == -1:
+        module_to_load = command_module
+    else:
+        module_to_load = command_module[:command_module.rfind('.')]
     import_module(module_to_load).load_params(command)
     _apply_parameter_info(command, command_table[command])
 
 
+def _get_command_table_from_extensions():
+    from azure.cli.core.extension import EXTENSIONS_MOD_PREFIX, get_extension_names, get_extension_path
+    extensions = get_extension_names()
+    if extensions:
+        logger.debug("Found {} extensions: {}".format(len(extensions), extensions))
+        for ext_name in extensions:
+            ext_dir = get_extension_path(ext_name)
+            sys.path.append(ext_dir)
+            pos_mods = [n for n in os.listdir(ext_dir) if n.startswith(EXTENSIONS_MOD_PREFIX)]
+            try:
+                assert len(pos_mods) == 1, \
```

Comment:
You know assert can be disabled at the interpreter level right? If this test is important, you should transform this with "if / raise".